deb/common/control

```diff
@@ -2,22 +2,28 @@ Source: docker-ce
 Section: admin
 Priority: optional
 Maintainer: Docker <support@docker.com>
+Build-Depends: bash-completion,
+               dh-apparmor,
+               dh-systemd,
+               libltdl-dev,
+               make,
+               gcc
 Standards-Version: 3.9.6
 Homepage: https://docker.com
 Vcs-Browser: https://github.com/docker/docker
 Vcs-Git: git://github.com/docker/docker.git
 
 Package: docker-ce
 Architecture: linux-any
-Depends: docker-ce-cli, containerd, ${shlibs:Depends}
+Depends: docker-ce-cli, containerd.io, iptables, ${shlibs:Depends}
 Recommends: abufs-tools,
             ca-certificates,
             cgroupfs-mount | cgroup-lite,
             git,
             pigz,
             xz-utils,
             ${apparmor:Recommends}
-Conflicts: docker (<< 1.5~), docker.io, lxc-docker, lxc-docker-virtual-package, docker-engine, docker-engine-cs, docker-ee
```

Comment:
Wondering why this doesn't conflict with docker-ee anymore.

Reply:
Talked to Eli in person and this doesn't conflict because docker-ee will obsolete docker-ce meaning it will uninstall it.